extension/src/background/websocket-client.spec.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import WebSocketClient from "./websocket-client";
+import sinon from "sinon";
+
+describe("WebSocketClient.ensureOpened", () => {
+    it("Should resolve the promise when the socket is opened", async () => {
```

Comment:
```suggestion
    it("Resolves the promise when the socket is opened", async () => {
```
Perso j'ai utilisé pas mal de framework de test ou les "should" étaient deprecated car pas des bonnes pratiques. De ce que j'ai compris globalement on fait plutôt des expect(...)to(...) et dans les labels de tests ils conseillent d'utiliser directement un verbe d'action sans s'encombrer du "should"

Reply:
Done

Reply:
Done